lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -466,16 +466,22 @@ def check_watched_item(self, job_state):
                 return None
             # there is no job responding to this job_id, it is either lost or something happened.
             log.error("No Jobs are available under expected selector app=%s", job_state.job_id)
-            with open(job_state.error_file, 'w') as error_file:
-                error_file.write("No Kubernetes Jobs are available under expected selector app=%s\n" % job_state.job_id)
-            self.mark_as_failed(job_state)
+            try:
```

Comment:
Good! this should at least avoid the missing file to derail the loop of jobs being checked.